code/modules/cargo/supplypod.dm

```diff
@@ -80,8 +81,35 @@
 	landingDelay = 20 //Very speedy!
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 
-/obj/structure/closet/supplypod/Initialize()
+/obj/structure/closet/supplypod/securitypod
+	style = STYLE_SECURITY
+	bluespace = TRUE
+	explosionSize = list(0,0,0,1)
+	var/obj/item/radio/Radio
```

Comment:
you should lowercase this var and also document it as required by documentation standards